e2e/tests/integrations/admin/create-integration.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Page, expect, test } from "@playwright/test";
+import { faker } from "@faker-js/faker";
+
+test.use({ storageState: "playwright/.auth/admin.json" });
+
+export const IntegrationTypes = {
+  SEARCH_API: "search-api",
+  ENTRY_API: "entry-api",
+  WIDGETS: "widgets",
+} as const;
```

Comment:
@simon-debruijn I see that these types already exist inside the app.
Is there an easy way to import them here?